docs/sources/alerting/manage-notifications/create-contact-point.md

```diff
@@ -0,0 +1,58 @@
+---
+aliases:
+  - /docs/grafana/latest/alerting/contact-points/create-contact-point/
+  - /docs/grafana/latest/alerting/contact-points/message-templating/
+  - /docs/grafana/latest/alerting/message-templating/
+  - /docs/grafana/latest/alerting/unified-alerting/message-templating/
+  - /docs/grafana/latest/alerting/contact-points/delete-contact-point/
+  - /docs/grafana/latest/alerting/contact-points/edit-contact-point/
+  - /docs/grafana/latest/alerting/contact-points/test-contact-point/
+  - /docs/grafana/latest/alerting/manage-notifications/create-contact-point/
+keywords:
+  - grafana
+  - alerting
+  - guide
+  - contact point
+  - templating
+title: Manage contact points
+weight: 100
+---
+
+# Manage contact points
+
+Use contact points to define how your contacts are notified when an alert rule fires. You can create, edit, delete, and test a contact point.
+
+To add a contact point:
+
+1. In the Grafana menu, click the **Alerting** (bell) icon to open the Alerting page listing existing alerts.
+1. Click **Contact points** to open the page listing existing contact points.
+1. Click **New contact point**.
+1. From the **Alertmanager** dropdown, select an Alertmanager. By default, Grafana Alertmanager is selected.
+1. In **Name**, enter a descriptive name for the contact point.
+1. From **Contact point type**, select a type and fill out mandatory fields. For example, if you choose email, enter the email addresses. Or if you choose Slack, enter the Slack channel(s) and users who should be contacted.
+1. Some contact point types, like email or webhook, have optional settings. In **Optional settings**, specify additional settings for the selected contact point type.
+1. In Notification settings, optionally select **Disable resolved message** if you do not want to be notified when an alert resolves.
+1. To add another contact point type, click **New contact point type** and repeat steps 6 through 8.
+1. Click **Save contact point** to save your changes.
+
+To edit a contact point:
```

Comment:
same here

Reply:
here's an example of how I format multi-task topics.
https://grafana.com/docs/grafana/latest/administration/team-management/